tests/foreman/ui/test_activationkey.py

```diff
@@ -206,6 +207,93 @@ def test_positive_create_with_envs(self):
                     )
                     self.assertIsNotNone(self.activationkey.search(name))
 
+    @run_only_on('sat')
+    @tier2
+    @upgrade
+    def test_positive_access_non_admin_user(self):
```

Comment:
Do you think its good to keep this test in test_activation_key. 
We can add generic test in test_filter for not only AK entity but for other entities too ?

Reply:
Yes, as BZ description explicitly speak about AK. For BZ that has customer tickets is not acceptable to make any form of generalizations. It should be exact step-by-step scenario